test/fail_compilation/iasm1.d

```diff
@@ -142,3 +142,21 @@ void test6()
     }
     L1:;
 }
+
+#line 700
+
+/* TEST_OUTPUT:
+---
+fail_compilation/iasm1.d(713): Error: bad type/size of operands `mov`
+---
+*/
+
+void foo()
```

Comment:
Am I missing something or this test is duplicated?

Reply:
One of them is 32 bit the other isn't

Reply:
> One of them is 32 bit the other isn't

You can add a permute args instead of duplicating the test case then.

Reply:
No I can't. One of them is in a file for 32 bit tests, the other one is in a file for 64 bit tests.

Reply:
Sure, makes sense, chopping it would be slower.